src/Symfony/Component/HttpClient/Tests/AsyncDecoratorTraitTest.php

```diff
@@ -76,7 +77,7 @@ public function testRetryTransportError()
             }
         });
 
-        $response = $client->request('GET', 'http://localhost:8057/chunked-broken');
+        $response = $client->request('GET', 'http://localhost:8058/');
```

Comment:
This defeats the purpose of the test case, which should be to retry a broken body.
Can't we assert something on the body instead?

Reply:
I tried but I couldn't make it work.
The problem is, when the TransportException (`Transfer closed with outstanding data remaining from chunked response`) is thrown, the first chunk is already yielded.
We can retry the request, but we cannot yield the first chunk of the second request.
Could you please provide some idea?

Reply:
I found a way, PR updated.